doc/release-notes-21392.md

```diff
@@ -0,0 +1,7 @@
+Low-level changes
+=================
+
+RPC
+---
+
+* BIP 9 has been replaced with a partial implementation of BIP 8. This change is reflected in `getblockchaininfo` where references to BIP 9 have been replaced with references to BIP 8.
```

Comment:
`getblockchaininfo` has a `bip9` field that is changed to `bip8`. This only effects signet and regtest though since those are the only ones that have taproot show up in `getblockchaininfo`.

But this point is moot as I have removed the rename commit.